Sources/RInAppMessaging/Utilities/BundleInfo.swift

```diff
@@ -44,7 +44,12 @@ internal extension Bundle {
         guard let sdkBundleURL = sdk?.resourceURL else {
             return nil
         }
-        return .init(url: sdkBundleURL.appendingPathComponent("RInAppMessagingResources.bundle"))
+        // in some cases `resourceURL` returns a path to .framework file and sometimes to the .bundle file inside
+        if sdkBundleURL.pathExtension == "bundle" {
+            return .init(url: sdkBundleURL)
+        } else {
+            return .init(url: sdkBundleURL.appendingPathComponent("RInAppMessagingResources.bundle"))
+        }
```

Comment:
You could use pattern matching here, maybe clearer, not mandatory:
```
switch sdkBundleURL.pathExtension {
    case "bundle":
            return .init(url: sdkBundleURL)
    default:
            return .init(url: sdkBundleURL.appendingPathComponent("RInAppMessagingResources.bundle"))
}
```

Reply:
@malayli Yeah it looks good for me but I think Sonar will complain about this:
https://rules.sonarsource.com/swift/RSPEC-1301